### ISSUE TITLE
Using zoom throws exception

### ISSUE THREAD

**Issue by sstainba** [vue-leaflet/vue-leaflet#244]
Vue 3.0.0
Quasar 2.6.0
Leaflet 1.9.2

Using the demo nearly verbatim, attempting to zoom throws an exception in setZoom of "Attempted to load in infinite number of tiles."

GridLayer.js:665 Uncaught (in promise) Error: Attempted to load an infinite number of tiles
    at Proxy._update (GridLayer.js:665:45)
    at Proxy._setView (GridLayer.js:574:10)
    at Proxy._resetView (GridLayer.js:530:8)
    at Proxy.fire (Events.js:195:9)
    at Proxy._move (Map.js:1231:10)
    at Proxy._resetView (Map.js:1186:5)
    at Proxy.setView (Map.js:206:8)
    at Proxy.setZoom (Map.js:218:15)
    at Object.setZoom (LMap.vue:309:34)
    at utils.js:32:19

```
<template>
  <l-map v-model="zoom"
         v-model:zoom="zoom"
         :center="center"
         style="height: 80vh">
    <l-tile-layer :url="url" />
    <l-control-layers/>
  </l-map>
</template>

<script>
  import "leaflet/dist/leaflet.css"
  import { LMap, LTileLayer, LControlLayers } from "@vue-leaflet/vue-leaflet";
  export default {
    name: 'LocationsPage',
    components: {
      LMap,
      LTileLayer,
      LControlLayers
    },
    data() {
      return {
        url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
        zoom: 15,
        center: [51.505, -0.159],
      }
    }
  }
</script>

```

**Comment by Vectorial1024** [comment]
This seems like a Leaflet-side problem, not this library's problem; see eg https://github.com/vue-leaflet/vue-leaflet/issues/238, https://github.com/vue-leaflet/vue-leaflet/issues/239#issuecomment-1292129070

**Comment by cenl-julien** [comment]
Hello,

After many tests, I found a solution. 

You must specify in l-map props "minZoom" and "maxZoom", it's works for me :

`<template>
	<div class="position-relative" style="height:60vh;">
		<l-map
			:zoom="zoom"
			:minZoom="3"
			:maxZoom="18"
			ref="map"
			style="height: 100%; width: 100%"
			:center="center"
			@move="doSomething"
			>

			<l-control-layers position="topleft"></l-control-layers>

			<l-tile-layer
	     		url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
	    	></l-tile-layer>
		</l-map>
	</div>
</template>

<script>

	import "leaflet/dist/leaflet.css";
	import { LMap, LControlLayers, LTileLayer } from "@vue-leaflet/vue-leaflet";

	export default{
		components: {
			LMap,
			LControlLayers,
			LTileLayer
		},

		data(){
			return{
				loading: true,
				zoom: 8,
				center: [48.6833, 6.2],
			}
		},
		methods:{
			doSomething: function(event){

			},
		},
	}
</script>`

**Comment by aentwist** [comment]
This is most likely an issue with how leaflet is being used. Probably, unintended usage here coupled with poor validation from leaflet.

See the leaflet issue that describes the unexpected usage https://github.com/Leaflet/Leaflet/issues/6416.

**Comment by mikeu** [comment]
Thank you all for reporting and investigating this issue. The problem was that we incorrectly used `undefined` as a default value intending to tell Leaflet that it should use its own default, without realising that it did indeed accept `undefined` as a value for the parameter despite its not being valid.

I've just released [v0.8.0](https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.8.0), which should resolve this issue. Please let me know if you still experience it!

**Comment by Abbos6465** [comment]
Raxmat
